Prague 2020 Schedule WIP

plaindocs opened the pull request:
- [x] add break after lightning talk
- [x] times++

plaindocs committed:
Schedule WIP

plaindocs committed:
EU times please

ericholscher commented on docs/_data/prague-2020-schedule.yaml:
```diff
@@ -0,0 +1,77 @@
+talks_day1:
+  - time: '8:00'
+    title: 'Platform opens'
+  - time: '9:00'
```
Normally we've started Prague at 10am, did we change this on purpose?

thatdocslady replied:
NOPE @plaindocs we need to shift everything forward by 1hr <3

plaindocs committed:
Errant US time

plaindocs committed:
Make it ALL LATER

plaindocs committed:
Make space for lightning talks

plaindocs commented:
@thatdocslady @ericholscher Times alatered, lighting talks given space. 
Please double check times, also the timing of the social.

thatdocslady commented on docs/_data/prague-2020-schedule.yaml:
```diff
@@ -1,77 +1,77 @@
+  - time: '9:00'
     title: 'Platform opens'
-  - time: '9:30'
+  - time: '10:30'
```
```suggestion
  - time: '10:00'
```

thatdocslady commented on docs/_data/prague-2020-schedule.yaml:
```diff
@@ -1,77 +1,77 @@
+  - time: '10:30'
     title: Morning introduction - Introduction to Sprints & Project Introductions
-  - time: '9:45'
+  - time: '10:45'
```
```suggestion
  - time: '10:15'
```

thatdocslady commented on docs/_data/prague-2020-schedule.yaml:
```diff
@@ -1,77 +1,77 @@
     title: Day 1 wraps up
-  - time: '18:00'
-    title: '<b>Social Event</b>'
   - time: '20:00'
```
```suggestion
  - time: '19:30'
```

thatdocslady commented on docs/_data/prague-2020-schedule.yaml:
```diff
@@ -1,77 +1,77 @@
-    title: '<b>Social Event</b>'
   - time: '20:00'
+    title: '<b>Social Event</b>'
+  - time: '22:00'
```
```suggestion
  - time: '21:30'
```

thatdocslady commented on docs/_data/prague-2020-schedule.yaml:
```diff
@@ -1,77 +1,77 @@
     slug: the-importance-of-using-analytics-and-feedback-for-your-documentation-karissa-van-baulen
-  - time: '9:45'
+  - time: '10:45'
     title: "<b>Job Fair Starts</b>"
```
```suggestion
    title: "<b>Job Fair Starts (open until lunch)</b>"
```

thatdocslady commented on docs/_data/prague-2020-schedule.yaml:
```diff
@@ -1,77 +1,77 @@
+  - time: '17:45'
     title: Closing address
-  - time: '17:00'
+  - time: '18:15'
```
```suggestion
  - time: '18:00'
```

thatdocslady reviewed (approved):
Fixed a couple of spots where we shortened the timings, otherwise 💯

plaindocs committed:
Update docs/_data/prague-2020-schedule.yaml

Co-authored-by: Mikey Ariel <thatdocslady@users.noreply.github.com>

plaindocs committed:
Update docs/_data/prague-2020-schedule.yaml

Co-authored-by: Mikey Ariel <thatdocslady@users.noreply.github.com>

plaindocs committed:
Update docs/_data/prague-2020-schedule.yaml

Co-authored-by: Mikey Ariel <thatdocslady@users.noreply.github.com>

plaindocs committed:
Update docs/_data/prague-2020-schedule.yaml

Co-authored-by: Mikey Ariel <thatdocslady@users.noreply.github.com>

plaindocs committed:
Update docs/_data/prague-2020-schedule.yaml

Co-authored-by: Mikey Ariel <thatdocslady@users.noreply.github.com>

plaindocs committed:
Update docs/_data/prague-2020-schedule.yaml

Co-authored-by: Mikey Ariel <thatdocslady@users.noreply.github.com>

ericholscher commented on docs/_data/prague-2020-schedule.yaml:
```diff
@@ -0,0 +1,77 @@
+  - time: '13:00'
+    slug: the-baseline-or-technical-writing-for-non-technical-readers-paul-brown
+  - time: '13:45'
+    title: Lunch (60 mins)
```
This feels a little late to me -- should we swap one talk after lunch, and do lunch at 13:00?

plaindocs replied:
Yeah, makes sense. :/

thatdocslady replied:
What about the return of the snack break? Maybe we can add 15min after the first 2 talks (well, 3 talks including the conference intro) and move lunch 15min later?

ericholscher replied:
That seems to move lunch later, not earlier? If anything -- we should add a snack break in the afternoon if we move lunch, since it will be a larger block of talks. That pushes the day back even more though, and we're already ending really late.

I was going to suggest doing 40 minute blocks instead of 45, since we've moved the Q&A and are doing live-streamed talks, which would tighten up the schedule nicely. I think it's probably worth doing, but annoying at this stage :/

thatdocslady replied:
Yeah, I think it's best to keep 45min slots for now coz it might mess up the schedule too much. Maybe bump one talk to after lunch and add the 15min snack break in the afternoon instead of the morning?

plaindocs replied:
Alright folks, lunch switched.
Snack break seems less important cos folks can just grab stuff from their own kitchen.
I'm calling this done.

plaindocs committed:
Add speaker image

plaindocs committed:
No late lunches

thatdocslady committed:
Prague 2020 schedule blog post (#1344)

* Scheduled weekly dependency update for week 38 (#1342)

* Update sphinx from 3.1.2 to 3.2.1
* Update ablog from 0.10.6 to 0.10.9
* Update tox from 3.17.1 to 3.20.0
* Update sphinx_autobuild from 0.7.1 to 2020.9.1
* Update sphinxcontrib.datatemplates from 0.6.1 to 0.7.1
* Update urllib3 from 1.25.9 to 1.25.10
* Update yamale from 3.0.1 to 3.0.4
* Update ruamel.yaml from 0.16.10 to 0.16.12
* Update video data template for sphinxcontrib-datatemplates 0.7

Co-authored-by: Sasha Romijn <github@mxsasha.eu>

* Add -j auto for parallelised sphinx builds (#1343)

* Adding additional bias resources (#1326)

Adding link to APA's bias free language article and the IETF's draft on terminology, power, and oppression

* Prague 2020 schedule blog post

Co-authored-by: pyup.io bot <github-bot@pyup.io>
Co-authored-by: Sasha Romijn <github@mxsasha.eu>
Co-authored-by: dwhitelf <53910321+dwhitelf@users.noreply.github.com>

plaindocs committed:
Merge branch 'master' into prague-2020-schedule

plaindocs commented:
Now with added blog post. Hat tip @thatdocslady
